css-ui-4/Overview.bs

```diff
@@ -140,196 +140,14 @@ which itself replaced and superseded the following:
 		<a href="https://www.w3.org/TR/2000/WD-css3-userint-20000216">User Interface for CSS3 (16 February 2000)</a> [[CSS-UI-3]]
 </ul>
 
+One feature previously defined in [[CSS-UI-3]] has been moved to [[CSS-SIZING-3]] rather than this specification.
```

Comment:
Delete this sentence and the next two headings, mark the following added text as a note, and add empty anchors for all the removed anchors, e.g. via <a id=".."></a> or somesuch. This way links won't be broken, but we also don't have to carry around an empty section.